src/Integrations/Corvus/SettingsCorvus.php

```diff
@@ -492,14 +511,30 @@ static function ($option) use ($selectedStoreId) {
 								[
 									'component' => 'checkboxes',
 									'checkboxesFieldLabel' => '',
-									'checkboxesName' => UtilsSettingsHelper::getSettingName(self::SETTINGS_CORVUS_IBAN_USE_KEY),
+									'checkboxesName' => UtilsSettingsHelper::getSettingName(self::SETTINGS_CORVUS_ENTRY_ID_USE_KEY),
 									'checkboxesContent' => [
 										[
 											'component' => 'checkbox',
-											'checkboxLabel' => \__('Use IBAN Payment', 'eightshift-forms'),
-											'checkboxHelp' => \__('To use IBAN Payment you must have this feature enabled in your Corvus account by contacting support.', 'eightshift-forms'),
-											'checkboxIsChecked' => UtilsSettingsHelper::isSettingCheckboxChecked(self::SETTINGS_CORVUS_IBAN_USE_KEY, self::SETTINGS_CORVUS_IBAN_USE_KEY, $formId),
-											'checkboxValue' => self::SETTINGS_CORVUS_IBAN_USE_KEY,
+											'checkboxLabel' => \__('Use Entry Id', 'eightshift-forms'),
+											'checkboxHelp' => \__('Use Entry Id instead of `increment ID` as Corvus `order_number` value. This is used if you want to refference the entry after the form submission. Make sure you have Entries feature turened `on`.', 'eightshift-forms'),
```

Comment:
```suggestion
											'checkboxHelp' => \__('Use Entry Id instead of `increment ID` as Corvus `order_number` value. This is used if you want to reference the entry after the form submission. Make sure you have Entries feature turned `on`.', 'eightshift-forms'),
```